docs/src/main/sphinx/connector/hudi.rst

```diff
@@ -157,3 +157,43 @@ Here are some sample queries:
     ------------+--------+
      2018-08-31 |  99  |
     (1 rows)
+
+Rolling back to a previous version
+-----------------------------------
+
+Hudi allows us to store multiple versions of data in the table overtime while
+providing ``snapshot isolation``. The number of versions are configurable and depends
+on how much tradeoff one prefers for space used by those older versions. Hence, as
+one can imagine, Hudi has the capabilities to allow to accessing the state of a table
+at a certain point/instant in time. This could be done by allowing read mechanisms
+from a Hudi table that go back and read the version of the data that was written a
+particular point/instant in time.
+
+For example, you could find the version IDs for the ``customer_orders`` table
+by running the following query::
+
+    SELECT _hoodie_commit_time
+    FROM example.testdb.customer_orders
+    ORDER BY _hoodie_commit_time DESC
```

Comment:
I feel recommending this query isn't a good idea.  This query causes full table scan and I suppose it means we can't time travel to where the operation was `DELETE` statement. It would be nice to provide a system table that provide commit history first in #16181